include/big_core_input.h

```diff
@@ -84,6 +86,7 @@ class BigCoreInput : public BigCoreBase
     //T* getTile(uint64_t imageNum, uint64_t tileNum = 0);
     // RV: Libore, Tvoje verze by musela alokovat nove pole, kam by ulozila data. A uzivatel by ho pak musel dealokovat.
     // RV: Radeji bych volil verzi, kam uz posles alokovane pole (a musis si ho po sobe i uklidit)
+	// LB: v obou pripadech tu pamet potom musi dealokovat, nevidim v tom rozdil
```

Comment:
Rozdil je jednak v tom, ze takto se da ta pamet vyuzit opakovane, takze to muze byt o neco rychlejsi. A take to uzivatele spise trkne, ze ma pamet uvolnit, kdyz si ji alokoval.